picard/formats/__init__.py

```diff
@@ -40,6 +40,11 @@ def supported_formats():
     return formats
 
 
+def supported_extensions():
+    extensions = [ext for exts, name in supported_formats() for ext in exts]
+    return extensions
```

Comment:
Huh? Not sure why there is 2 `for` but i think the following should give the same result:

```python
return [ext for ext, _ in supported_formats()]
```

Also no need of intermediate variable.